package.json

```diff
@@ -29,9 +29,11 @@
 	},
 	"dependencies": {
 		"axios": "^1.2.3",
+		"body-parser": "^1.20.1",
```

Comment:
BodyParser is included in Express since a while back. You can remove this dependency, change the `bodyparser.json()` line to `express.json()` and it will work just the same.

Reply:
THANK YOU so much for that. Now I know!